lib/alarm.js

```diff
@@ -30,6 +30,7 @@ var util = require('util'),
 function raise(alarm, context, message) {
     var state = alarms[alarm];
     context = (process.domain && process.domain.context) || {};
+
```

Comment:
CHANGES_NEXT_RELEASE entries should be added regarding the changes in this documentation. Suggestion:

```
 - Add: /api-docs endpoint providing swagger-based documentation of the HTTP endpoints exposed by Perseo FE
- Hardening: software quality improvement based on ISO25010 recommendations 
```

Reply:
Fixed in 210813ba445911619199c59d729c8b724f5aef5a